examples/default/Makefile

```diff
@@ -72,14 +72,14 @@ endif
 include $(RIOTBASE)/Makefile.include
 
 # Set a custom channel if needed
-ifeq (,$(filter cc110x,$(USEMODULE)))     # radio is cc110x sub-GHz
+ifneq (,$(filter cc110x,$(USEMODULE)))          # radio is cc110x sub-GHz
```

Comment:
@miri64, thinking of it : would it make sense to move this block somewhere in [Makefile.include](https://github.com/RIOT-OS/RIOT/blob/master/Makefile.include), maybe [here](https://github.com/RIOT-OS/RIOT/blob/master/Makefile.include#L206)

Then no need to duplicate this code in all applications ?